core/src/test/java/io/kestra/core/runners/pebble/filters/StringFilter.java

```diff
@@ -0,0 +1,50 @@
+package io.kestra.core.runners.pebble.filters;
+
+import io.pebbletemplates.pebble.error.PebbleException;
+import io.pebbletemplates.pebble.extension.Filter;
+import io.pebbletemplates.pebble.template.EvaluationContext;
+import io.pebbletemplates.pebble.template.PebbleTemplate;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.util.List;
+import java.util.Map;
+
+public class StringFilter implements Filter {
```

Comment:
As Shruti said, this is a duplicate of the class, please remove this file and implement a proper test